test/functional/wallet_multiwallet.py

```diff
@@ -211,5 +211,15 @@ def run_test(self):
         # Fail to load if wallet file is a symlink
         assert_raises_rpc_error(-4, "Wallet file verification failed: Invalid -wallet path 'w8_symlink'", self.nodes[0].loadwallet, 'w8_symlink')
 
+        self.log.info("Test dynamic wallet creation.")
+
+        # Fail to create a wallet if it already exists.
+        assert_raises_rpc_error(-4, "Wallet w2 already exists.", self.nodes[0].createwallet, 'w2')
+
+        # Successfully create a wallet with a new name
+        self.nodes[0].createwallet('w9')
+
+        assert 'w9' in self.nodes[0].listwallets()
+
```

Comment:
Also call some method in `w9`, like `get_wallet_rpc('w9').getwalletinfo()`?

Reply:
sure. Added